client/src/app/views/login.js

```diff
@@ -0,0 +1,32 @@
+window.LoginView = Backbone.View.extend({
+  initialize: function() {
+    this.render();
+  },
+  events: {
+    'submit' : 'loginUser'
+  },
+  render: function() {
+    $(this.el).html(this.template());
+    return this;
+  },
+
+  loginUser: function(e){
+    e && e.preventDefault();
+    var $formUsername = this.$el.find('form .username');
+    var $formPassword = this.$el.find('form .password');
+    var user = new User({username: $formUsername.val(), password: $formPassword.val()});
+    var userJSON = {username: $formUsername.val(), password : $formPassword.val()}
+    var _this = this;
+    $.ajax({
```

Comment:
@d4ncer @ekeric13 this shouldn't be happening in the view — hitting the server endpoint is a model concern